src/languages/en.ts

```diff
@@ -1871,6 +1871,12 @@ export default {
                 valuePercentageRange: 'Please enter a valid percentage between 0 and 100.',
                 genericFailureMessage: 'An error occurred while updating the tax rate, please try again.',
             },
+            deleteTaxConfirmation: 'Are you sure you want to delete this tax?',
```

Comment:
missing es translations?

Reply:
Asked for translations [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1710843079162819).